tests/components/elevenlabs/conftest.py

```diff
@@ -44,6 +45,14 @@ def mock_async_client() -> Generator[AsyncMock]:
             "homeassistant.components.elevenlabs.config_flow.AsyncElevenLabs",
             new=mock_async_client,
         ),
+        patch(
+            "homeassistant.components.elevenlabs.__init__.AsyncElevenLabs",
```

Comment:
This is not a valid path

Reply:
Apologies, removed as it was also not required. All tests passed with it removed.